README.md

```diff
@@ -79,8 +79,8 @@ Important Note: This standard for crypto wallets security is product of ongoing
 
 ### 6. Communication
 
-|  #  | Description                                                                                                                                                                                                                                                                      | Comment            |
-| --- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+|  #  | Description                                                                                                                                                                                                                                                                      | Comment            | Attacks/Reports |
```

Comment:
Can you add the attacks/reports column to section #5? I've a report related to 5.2